metadata.json

```
@@ -14,7 +14,7 @@
     },
     {
       "name": "puppetlabs-apt",
-      "version_requirement": ">= 9.2.0 < 10.0.0"
+      "version_requirement": ">= 9.2.0 <= 10.0.0"
```

```suggestion
      "version_requirement": ">= 9.2.0 < 11.0.0"
```

I think it is usually done like this to allow 10.x.x, the latest version is 10.0.1 and would not fit "<= 10.0.0".